src/components/drop/redux/DropReducers.ts

```diff
@@ -0,0 +1,33 @@
+import {IReactVaporState} from '../../../ReactVapor';
+import {IReduxAction} from '../../../utils/ReduxUtils';
+import {DropReducerActions, IDropPayload} from './DropActions';
+
+export interface IDropCompositeState extends IDropPayload {}
+
+export const dropInitialState: IDropCompositeState = {id: undefined, isOpen: false};
+
+export const dropCompositeReducer = (

```

Comment:
I think `composite` doesn't add any value to our understanding, it should simply be named `dropReducer`